controllers/uaa.go

```diff
@@ -49,3 +98,476 @@ func (c *UAAContext) UaaInfo(rw web.ResponseWriter, req *web.Request) {
 		rw.Write([]byte("{\"status\": \"Bad request\", \"error_description\": \"Missing valid guid.\"}"))
 	}
 }
+
+// InviteEmail provides inviteTextTpl and inviteEmailTpl the value to access invite_url query
+type InviteEmail struct {
+	URL string
+}
+
+const inviteTextTpl = `
```

Comment:
@jcscottiii This is being used in the "text" content: https://github.com/18F/cg-dashboard/pull/1086/files#diff-f800b1e943e88b0a2d3081775265a1f2R56